docs/deploying.md

```diff
@@ -90,6 +96,48 @@ kustomize build $BMOPATH/ironic-deployment/default | kubectl apply -f-
 
 where $BMOPATH points to the baremetal-operator path.
 
+### Deploying with TLS
+
+For this, you need to first copy your Ironic CA Certificate, then build the
```

Comment:
Is the goal to allow a deployment script (or helm chart or whatever) to create the files?

Reply:
In our case, we want to be able to provide the CA for the ironic deployment. Ironic itself would use the certificates generated by cert-manager, but for BMO to accept them all, we need to use the CA, not a specific certificate.
@dhellmann yes, at the end it will be cert-manager creating the certificates and keys, however, it could be given the CA to do so.

Reply:
> we need to use the CA, not a specific certificate

This is not really correct: in the simplest case you can use the certificate. I don't disagree there is a valid case for having a CA, but I would prefer that we also support a simple one-knob configuration "get_me_tls: true" where we'll just generate a pair key-certificate and pass certificate as both CERT_FILE and CACERT_FILE.

Reply:
ok, this is now supported, we can just generate a certificate and a key, and pass it to both ironic and BMO and get it going by passing the same file for IRONIC_CACERT_FILE in BMO.

Reply:
Let's mention that the CA certificate is no longer required?